channels/drive/client/drive_file.c

```diff
@@ -563,6 +569,8 @@ BOOL drive_file_set_information(DRIVE_FILE* file, UINT32 FsInformationClass, UIN
 				if (m != st.st_mode)
 					fchmod(file->fd, st.st_mode);
```

Comment:
Implemented winpr function for this with #3164 (granted, works on filename, but we have that)